bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -115,6 +116,13 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     public void initialize() {
         thingConfig = getConfigAs(SomfyTahomaConfig.class);
 
+        try {
+            httpClient.start();
+        } catch (Exception e) {
```

Comment:
Can you please specify the exceptions you expect to catch here.

Reply:
Sorry, the exception is generic one...

protected void start​(LifeCycle l) throws java.lang.Exception
Starts the given lifecycle.
Parameters:
l - the lifecycle to start
Throws:
java.lang.Exception - if unable to start lifecycle